src/main/java/uk/gov/service/notify/SendEmailResponse.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.service.notify;
+
+import org.json.JSONObject;
+
+import java.util.Optional;
+import java.util.UUID;
+
+public class SendEmailResponse {
```

Comment:
should these extend a `NotificationResponse` that defines all the shared fields? template, body, reference, notificationId, etc.